packages/blank-theme/src/components/breadcrumbs/index.tsx

```diff
@@ -0,0 +1,60 @@
+"use client"
+import { hexToRGBA } from "../../lib/hex-to-rgba"
+import { Stack, Typography } from "@mui/material"
+import { useTheme } from "next-themes"
+
+export default function BreadCrumbs({
+   title,
```

Comment:
where is components props passing rules, I want to match every props with rules

Reply:
@zelal-dev vai, I didn't work with this `blank theme`. Fixing this issue will be a different task.